app/models/valid_fulfilled_at_date.rb

```diff
@@ -0,0 +1,56 @@
+class ValidFulfilledAtDate
+
+  include ActiveModel::Validations
+  include DateHelper
+
+  validate :valid_format
+  validate :not_in_future, if: :to_time
+  validate :recent, if: :to_time
+
+  def self.min
+    # Beginning of previous fiscal year
+    SettingsHelper.fiscal_year_beginning(1.year.ago)
+  end
+
+  def self.max
+    Time.current
+  end
+
+  def initialize(string)
+    @string = string
+  end
+
+  def error
+    errors[:base].first
+  end
+
+  def to_s
+    @string
+  end
+
+  # Returns nil if the date is invalid
+  def to_time
+    time = parse_usa_date(@string).try(:to_date)
+    time.beginning_of_day + 12.hours if time.present?
+  end
+  alias presence to_time
+  # This allows us to set `fulfilled_at = ValidFulfilledAtDate.new("XX/XX/XXXX")`
```

Comment:
I am a little confused by what's going on here. Is the "this" you're referring to for `presence`, or `in_time_zone`?